apps/prairielearn/src/pages/instructorQuestionStatistics/instructorQuestionStatistics.ejs

```diff
@@ -2,7 +2,6 @@
 <html lang="en">
   <head>
     <%- include('../partials/head', {pageNote: question.qid}); %>
-    <%- include('../partials/mathjax'); %>
```

Comment:
I don't see anywhere in this page where MathJax is actually used.

Reply:
Lines like these get incorrectly copied around a lot when making new pages, yeah. Thanks for cleaning it up!